src/libs/PaymentUtils.js

```diff
@@ -14,10 +14,15 @@ import * as Localize from './Localize';
  * @returns {Boolean}
  */
 function hasExpensifyPaymentMethod(cardList = [], bankAccountList = []) {
-    return _.some(cardList, card => card) || _.some(bankAccountList, (bankAccountJSON) => {
+    const validBankAccount = _.some(bankAccountList, (bankAccountJSON) => {
         const bankAccount = new BankAccount(bankAccountJSON);
         return bankAccount.isDefaultCredit();
     });
+
+    // Hide any billing cards that are not P2P debit cards for now because you cannot make them your default method, or delete them
+    const validDebitCard = _.some(cardList, card => lodashGet(card, 'additionalData.isP2PDebitCard', false));
```

Comment:
```suggestion
    const validDebitCard = _.some(cardList, card => lodashGet(card, 'additionalData.isP2PDebitCard', true));
```

Don't we want to find any debit cards that has `isP2PDebitCard` set to true?

Reply:
Yep! I thought the third argument for lodashGet would be the default value if the path resulted as unresolved

Reply:
Yeah I think this is correct